go/test/endtoend/vtgate/schematracker/loadkeyspace/schema_load_keyspace_test.go

```diff
@@ -76,18 +78,37 @@ func TestLoadKeyspaceWithNoTablet(t *testing.T) {
 	for _, vttablet := range clusterInstance.Keyspaces[0].Shards[0].Vttablets {
 		err = vttablet.VttabletProcess.TearDown()
 		require.NoError(t, err)
+		timeout := time.After(1 * time.Minute)
```

Comment:
nitpick: I see the chance of extracting this as a method. you are using the same pattern twice in this file

Reply:
and again in `schema_restart_test.go`

Reply:
Fixed via 013d4047e5